app/src/main/java/org/sopt/cgv/feature/seats/component/SeatSelectionModal1.kt

```diff
@@ -0,0 +1,136 @@
+package org.sopt.cgv.feature.seats.component
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.persistentListOf
+import org.sopt.cgv.core.designsystem.component.button.CgvButton
+import org.sopt.cgv.core.designsystem.theme.Black
+import org.sopt.cgv.core.designsystem.theme.CGVTheme
+import org.sopt.cgv.core.designsystem.theme.PrimaryRed400
+import org.sopt.cgv.core.designsystem.theme.White
+import androidx.compose.material3.ExperimentalMaterial3Api
+
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SeatSelectionModal1(
+    modifier: Modifier = Modifier,
+    viewModel: SeatSelectViewModel,
```

Comment:
뷰모델 의존성을 제거해주세요!

```suggestion
    stepperValues: Map<String, Int>,
    onStepperIncrease: (String) -> Unit,
    onStepperDecrease: (String) -> Unit,
```
ViewModel을 직접 주입하는 대신 필요한 데이터와 이벤트 핸들러만 전달받도록 변경해주세요!
이렇게 바꾼다면 컴포저블의 재사용성을 높일 수 있을 거 같습니당